packages/content-processor/src/utils/ogp-fetcher.ts

```
@@ -0,0 +1,330 @@
+
+  // Fallback patterns for standard meta tags
+  const fallbackPatterns = {
+    title: /<title[^>]*>([^<]*?)<\/title>/i,
+    description: /<meta\s+[^>]*name\s*=\s*["']description["'][^>]*content\s*=\s*["']([^"']*?)["']/i,
+  };
+
+  // Try OGP patterns first
+  for (const [key, pattern] of Object.entries(ogpPatterns)) {
+    const match = html.match(pattern);
+    if (match && match[1]) {
+      (metadata as any)[key] = decodeHtmlEntities(match[1].trim());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)